core/include/usModuleRegistry.h

```diff
@@ -27,9 +27,11 @@
 #include <string>
```

Comment:
I believe the file can now be made private (usModuleRegistry_p.h) because we should be able to access all of the information via the new Framework class, right?